src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    private static final int SUPPLY_INDEX = 0;
+    private static final int BUY_INDEX = 1;
+    private static final String SPLIT_REGEX = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] sumSupplyAndBuy = new int[2];
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
```

Comment:
go through common mistakes and check yourself carefully
you have mistakes from that list